.github/workflows/nightly.yml

```diff
@@ -17,9 +17,6 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-      # with-ref part to be removed after merging 'adapters' branch with 'main'
```

Comment:
There is something similar in the e2e nightly which probably needs to be removed as well

Reply:
I'll take a look

Reply:
Removed.